examples/if_ls_glob.bats

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bats
+
```

Comment:
could you please add a description that this is an example file

Reply:
hmm... I think the file should be self-documenting after the latest refactors. I try to avoid comments when the code can self-document.